app/views/posts/index.html.erb

```diff
@@ -1,42 +1,79 @@
-<h1>Posts</h1>
+<div class="container"
+  <h1>Posts</h1>
 
-<%= form_for @post, url: {controller: "posts", action: "create"} do |form| %>
-  <%= form.label :message, "Create a new post" %>
-  <%= form.text_area :message, :rows =>4, :cols => 50 %>
-  <%= form.submit "Submit" %>
-<% end %>
+  <%= form_for @post, url: {controller: "posts", action: "create"} do |form| %>
+    <%= form.label :message, "Create a new post" %>
+    <%= form.text_area :message, :rows =>4, :cols => 50 %>
+    <%= form.submit "Submit" %>
+  <% end %>
 
-<% @posts.each do |post| %>
-  <div id='post-<%= post.id %>' class="post">
-    <div class="post__head">
-      <span class="post__head__user">
-        <%= post.user_name %>
-      </span>
-      <span class="post__head__date">
-        posted at: <%= post.posted_at %>
-      </span>
-      <br>
-    </div>
-    <div class="post__body">
-      <%= simple_format(post.message)%>
-      	<%= form_for(@like, url: post_likes_path(post)) do |form| %>
-      	  <%= form.hidden_field :post_id, :value => post.id %>
-          <%= form.hidden_field :user_id, :value => current_user.id %>
-      	  <%= form.submit "Like" %>
-      	<% end %>
-      <div id='likes'>likes: <%= Like.where(post_id: post.id).count %></div>
-    </div>
-    <div class="comment__body">
-      <% post.comments.each do |comment| %>
-        (user posting comment goes here) <%= simple_format(comment.message) %>
+  <% @posts.each do |post| %>
+    <div id='post-<%= post.id %>' class="post card">
+      <div class="post__head card-header">
+        <span class="post__head__user">
+          <%= post.user_name %>
+        </span>
+        <span class="post__head__date">
+          posted at: <%= post.posted_at %>
+        </span>
+        <br>
+      </div>
+      <div class="post__body card-block">
+        <div class="card-text">
+          <h6><%= post.message %></h6>
+        </div>
+      </div>
+
+      <% unless post.comments.count == 0 && Like.where(post_id: post.id).count == 0 %>
```

Comment:
move logic to the model in this if-else statement and test it

Reply:
done!